dojo-dungeon/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 
 
 dependencies {
-    api project(':dungeon')
+    implementation project(':dungeon')
```

Comment:
```suggestion
    api project(':dungeon')
```

Reply:
@tgrothe So nett und sinnvoll ich diese Änderung an sich finde, sie hat NICHTS in diesem PR zu suchen. (Wieder mal!)